go/vt/mysqlctl/schema.go

```diff
@@ -32,7 +32,7 @@ import (
 	tabletmanagerdatapb "vitess.io/vitess/go/vt/proto/tabletmanagerdata"
 )
 
-var autoIncr = regexp.MustCompile(" AUTO_INCREMENT=\\d+")
+var autoIncr = regexp.MustCompile(` AUTO_INCREMENT=\\d+`)
```

Comment:
Seems like this makes the regex incorrect? The backslash shouldn't need to be escaped with backticks.

Reply:
Fixed.